test/jdk/java/lang/reflect/AccessFlag/SuperAccessFlagTest.java

```diff
@@ -25,6 +25,7 @@
  * @test
  * @bug 8301720
  * @summary Test expected value of SUPER AccessFlag for pre-ValueClass .class file
+ * @requires !java.enablePreview
```

Comment:
The reasons for `@requires !java.enablePreview` is clear because the summary mentions `pre-ValueClass`.